From a350852845a4427ec09813b8b9e9242a1d30976f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Wed, 1 Mar 2023 02:42:56 +0100
Subject: [PATCH] Fix variable name for OpenGL library

In case Qt is build with GLES libraries, it does not implicitly add
the OpenGL library, and Quarter is not linked to the GL library.
---
 src/Gui/Quarter/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Gui/Quarter/CMakeLists.txt b/src/Gui/Quarter/CMakeLists.txt
index 45ee147fdd..e840e151d0 100644
--- a/src/Gui/Quarter/CMakeLists.txt
+++ b/src/Gui/Quarter/CMakeLists.txt
@@ -30,7 +30,7 @@ IF (Spacenav_FOUND)
 ENDIF(Spacenav_FOUND)
 
 
-SET(AllLibs ${Coin_LIBRARY} ${QT_LIBRARIES} ${OPENGL_LIBRARY} ${Extra_libraries})
+SET(AllLibs ${Coin_LIBRARY} ${QT_LIBRARIES} ${OPENGL_gl_LIBRARY} ${Extra_libraries})
 
 
 
-- 
2.39.1

